api/scripts/owned_tokens.py

```diff
@@ -0,0 +1,56 @@
+import os
```

Comment:
why is the other file deleted?

Reply:
Oh I thought it was better cause it's the file we are working on in the other or

Reply:
this is the file you are updating in the other pr right?

Reply:
you should just undo all changes to that file in this PR